### PR TITLE
Fixed README.md

### PR DESCRIPTION
**Pull request by eltontiet**

### CODE REVIEW

**Comment by eltontiet** [comment]
Please review the code soon, this is an urgent change

**Comment by ShakeebZ** [comment]
Implemented hyperlinks and removed User comments

File: README.md [hunk 1]
```diff
@@ -3,15 +3,3 @@
 This is an algorithm for the [House Prices - Advanced Regression Techniques competition on Kaggle](https://www.kaggle.com/c/house-prices-advanced-regression-techniques).
 I used Kaggle's "Intro to Machine Learning" course as a core supplemented by various online resources to teach myself
 the basics of machine learning and data analysis as well as apply them in an easy to see and evaluate manner.
-
-## User Comments:
```
Comment (eltontiet) [inline README.md]: Why did you remove this? This seems like a good feature for our customers

**Review by eltontiet — COMMENTED** [review]
Look at this

File: README.md (hunk 1 above)
Comment (ShakeebZ) [reply README.md]: Please view the lines below.
